operations/accountnotes.md

```diff
@@ -0,0 +1,345 @@
+# Account notes
+
+## Get all account notes
+
+Returns all account notes of an account (in the given activity state), optionally filtered by account identifiers by specific account note identifiers or other filter parameters.
+Note this operation uses [Pagination](../guidelines/pagination.md) and supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/accountNotes/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ChainIds": [
+        "1df21f06-0cfc-4960-9c58-a3bf1261663e",
+        "5fcd1933-22f2-40b9-84da-7db04cbecec2"
+    ],
+    "AccountNoteIds": [
+        "3ed9e2f3-4bba-4df6-8d41-ab1b009b6425",
+        "8a98965a-7c03-48a1-a28c-ab1b009b53c8"
+    ],
+    "AccountIds": [
+        "a6738390-c241-45b7-8e46-14f47207abe5",
+        "435d4d5f-d14f-48dc-a47e-0481fc28ead0"
+    ],
+    "UpdatedUtc": {
+        "StartUtc": "2022-10-10T00:00:00Z",
+        "EndUtc": "2022-10-17T00:00:00Z"
+    },
+    "ActivityStates": [
+        "Active"
+    ],
+    "Limitation":{
+        "Cursor": "e7f26210-10e7-462e-9da8-ae8300be8ab7",
+        "Count": 100
+    }
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ChainIds` | array of string | optional, max 1000 items | Unique identifiers of the chain. If not specified, the operation returns data for all chains within scope of the Access Token. |
+| `AccountNoteIds` | array of string | required, max 1000 items | Unique identifiers of [Account note](#account-note). |
```

Comment:
Is `AccountNoteIds` **required**, as it states here? This is not what it says in the description for the operation.